leakpro/attacks/mia_attacks/abstract_mia.py

```diff
@@ -256,3 +261,117 @@ def run_attack(self:Self) -> Union[AttackResult, List[AttackResult]]:
 
         """
         pass
+
+    def _memorization(self:Self) -> None:
+        """Run memorization score enhancement.
+
+        Memorization enhances the attack performance by only inlucing vulnerable data points
+
+        """
+
+        self.logger.info("Preparing memorization")
+        if not self.online:
+            self.logger.info("Using the offline version of the attack we make some assumptions in the absence of IN-models")
```

Comment:
be more explicit. "Some assumption"?

Reply:
Also, we dont have anything for offline? Create issue!

Reply:
Yes these assuptions should be explained. Should that maybe be done in the overleaf document maybe? And only have the offline implementation done in the file?

We have it for offline now, created an issue and resolved it in this PR

Reply:
I think we can have both. Yea, let's write it in the overleaf. Eventually, we also need to have a wiki or something in the github repo but thats for later